pkg/apis/application/v1alpha1/types.go

```diff
@@ -443,7 +443,7 @@ type ComponentParameter struct {
 }
 
 // SyncStatusCode is a type which represents possible comparison results
-type SyncStatusCode string
+type SyncStatusCode = string
```

Comment:
I added the equal sign because I was getting the following type error when comparing a `string` to the `SyncStatusCode` type:
![image](https://user-images.githubusercontent.com/614838/56158388-b3460180-5f76-11e9-8a1e-08d7f4380d3b.png)

I decided to change the type definition to a type alias because all of the possible members are strings and from a quick look at the `types.go` file it seems to be the convention (e.g., `type ApplicationConditionType = string`, `type HealthStatusCode = string`, etc.)

Reply:
Okay sounds good